src/artifacts-helper/README.md

```diff
@@ -32,7 +33,7 @@ Configures Codespace to authenticate with Azure Artifact feeds
 - `ms-codespaces-tools.ado-codespaces-auth`
 
 This installs [Azure Artifacts Credential Provider](https://github.com/microsoft/artifacts-credprovider)
-and optionally configures an alias for `dotnet`, `nuget`, `npm`, `yarn`, and `rush` that dynamically sets an authentication token
+and optionally configures functions which shadow `dotnet`, `nuget`, `npm`, `yarn`, and `rush` which dynamically sets an authentication token
```

Comment:
Please make this change in NOTES.md.  README is actually generated from the JSON and NOTES.md so does not need to be modified or included in the PR

Reply:
Oh good to know - updated